### DISCUSSION TITLE
Advantage of having HomeKit VWsFriend

### DISCUSSION THREAD

**Discussion by Julien4135** [tillsteinbach/VWsFriend#136] [Show and tell]
I just noticed, by handling the socket on the homekit, turning it on activates the car online without activating the air conditioning.
It is a success because currently even Volkswagen and WeConnect ID, cannot put the car online (activation of the preheating of the batteries) without putting the air conditioning in addition!
Congratulations !

![image](https://user-images.githubusercontent.com/93113346/148050863-fd2f87df-77cf-4f0c-8377-80f15e58b852.png)

**Comment by tillsteinbach** [comment]
Nice, but are you sure it is preheating the batteries when waking up only?

**Reply by Julien4135** [reply]
Yes I am sure I am sure by looking at the graffiti it is really sure!

![image](https://user-images.githubusercontent.com/93113346/148053583-d220c3cf-dbe8-43fa-b06d-ba195a98c003.png)

**Reply by Julien4135** [reply]
it is not significant on the graph because I got in the car to test something else. but I'm pretty sure it's true what I'm saying.

**Reply by Julien4135** [reply]
So I confirm my statements, this morning the car went online around 5:01 am I left for work around 5:15 am the battery last night was 230Km and this morning starting 239km and it went beyond (almost 245) to the first start because I have to stop to close the garage door so I gained 9km just by putting the car online, on the other hand what advances me to confirm that I am preheating the battery function is that when you click on the socket from HomeKit the car goes online some not even a minute later and you can clearly see the battery preheating symbol light up on the dashboard.
Here is for my return;)

![6](https://user-images.githubusercontent.com/93113346/148390635-a5822a08-0bb7-42bc-bdb5-f24a4d918779.png)

![image](https://user-images.githubusercontent.com/93113346/148390810-de171a87-bcb0-462e-8c92-ab7e65d8ad2c.png)

**Reply by tillsteinbach** [reply]
How does the preheating symbol look like in the dashboard?

**Reply by Julien4135** [reply]
![image](https://user-images.githubusercontent.com/93113346/148677201-06cf9e94-3b40-47f6-9269-31dcb81c401a.png)

**Reply by tillsteinbach** [reply]
Ah, this is the symbol for the cabin preheating.

**Reply by Julien4135** [reply]
Yes exactly, it is the symbol of the stationary air conditioning, except that at this moment the air conditioning of the vehicle is not in service, In fact nothing is specified on the preheating of the car on the manual in connection with this logo, But it is It's like getting in the car and switching on the vehicle and waiting for the system to slowly pump current from the battery to heat the batteries rather than too quickly going straight out onto the road in cold weather (that's a bit the same for thermal vehicles at the end)

So by pressing the button on the HomeKit application socket, it is possible to put the car online at a remote location even outside the house and the local network, and by deduction to activate the preheating of the batteries without activating the air conditioning in more.

**Reply by Julien4135** [reply]
![image](https://user-images.githubusercontent.com/93113346/148678766-22b789f3-6080-40bf-ad42-5b3b82f080fc.png)

**Reply by Julien4135** [reply]
What do you think, suddenly ?

**Reply by tillsteinbach** [reply]
I‘m still not 100% sure, but I will try to confirm your observations when it is really cold again here.

**Reply by Julien4135** [reply]
Ah but that's cool so you also have an ID.3 in your car garage, I didn't think so!
Looking forward to discussing the car elsewhere than here!

**Reply by tillsteinbach** [reply]
Yes it is an awesome car. Looking forward to software 3.0 where it will be hopefully on par with my Passat regarding the connectivity.

**Reply by Julien4135** [reply]
Passat GTE hybride ?

**Reply by Julien4135** [reply]
For the update, from what I understood is ultimately not as Volkswagen had said after update 2.3 Apparently the 2.4 update should be done in the workshop with several other physical modifications on the car, and then 3.0 will be made by areas like 2.3

**Reply by tillsteinbach** [reply]
yes GTE hybrid.
I don't expect to get 3.0 before april, but I think we will see new cars soon with 3.0 preinstalled.

**Reply by Julien4135** [reply]
Yes like the ID.5 which should be released soon when the 3.0 update is ready! It's a bit like at Apple I have the impression! Lol!

**Reply by Julien4135** [reply]
Class for the Passat !!! I have a Tiguan Hybrid from 2021 but it is less stylish and more in repair because module problem on the batteries

**Reply by Julien4135** [reply]
But it's a shame that WeConnectID is different than Weconnect I have seen some great useful functions that we must cry to the developers of the ID to hope to have them one day on the ID.3

**Reply by tillsteinbach** [reply]
The features of the WeConnect App will come. Let's take for example the window and door status. The WeConnect ID app can already display those even with the correct images for the ID. It is just that the car does not yet report those states.

**Reply by Julien4135** [reply]
We must at least have this option in early 2022 as he announced, well I think it's more in June 2022 for us but it should come out that's for sure!

**Reply by Julien4135** [reply]
Salvation ! I hope you have been well since then? For my part, I picked up my hybrid Tiguan from the garage 4 months ago because of a battery problem.
I would like to know how to have a second car in VWsfriends if possible? Could you give me the steps to follow please?

**Comment by tillsteinbach** [comment]
You just have to add it to your WeConnect account. Then it will appear in VWsFriend too:
<img width="984" alt="Bildschirmfoto 2022-03-11 um 08 10 02" src="https://user-images.githubusercontent.com/3825852/157819565-69882398-0313-44d3-81c2-e600366565b8.png">
<img width="997" alt="Bildschirmfoto 2022-03-11 um 08 10 35" src="https://user-images.githubusercontent.com/3825852/157819594-1ecde03d-0b79-4150-a426-ea3dda3d3ed1.png">

**Reply by Julien4135** [reply]
Ok thank you very much, the concern for me is that I don't have the same account, I use my wife's with a different email address.

**Comment by tillsteinbach** [comment]
What should work is that your account is added as a second user to the car

**Reply by Julien4135** [reply]
I will try like this. thank you :)
